AmazonChimeSDK/AmazonChimeSDK/audiovideo/DefaultAudioVideoController.swift

```diff
@@ -39,11 +39,6 @@ import Foundation
     }
 
     public func start(callKitEnabled: Bool) throws {
-        let audioPermissionStatus = AVAudioSession.sharedInstance().recordPermission
-        if audioPermissionStatus == .denied || audioPermissionStatus == .undetermined {
-            throw PermissionError.audioPermissionError
-        }
```

Comment:
I think technically this is a breaking change. I understand that we check audio permission again in `DefaultAudioClientController.start()`, but what if the builder is using its own `AudioClientController` instead of `DefaultAudioClientController`? Can we safely assume that the permission check will always be performed in `AudioClientController`?

Reply:
IMO `DefaultAudioClientController` at the lower level should be the correct and only place to check audio permission, and our "Default" implementations should demonstrate the best practice. I can mention in changelog it's a breaking change for developers who use their own `AudioClientController` implementation with no audio permission check.